authority-portal-frontend/src/app/pages/registration-pages/organization-onboard-page/organization-onboard-page/organization-onboard-page.component.html

```diff
@@ -106,7 +106,11 @@ <h2 class="wizard-step-title">Organization Profile</h2>
         <ng-template matStepLabel>Confirmation</ng-template>
         <div class="my-12 flex flex-col justify-center items-center">
           <h3 class="wizard-end-title text-center">
-            Thank you for onboarding on the MDS Portal!
+            {{
+              ('mds-id' | isActiveFeature)
+                ? 'Thank you for onboarding on the MDS Portal!'
+                : 'Thank you for onboarding on our Portal!'
```

Comment:
should the portal owner name be part of some config?

Reply:
I'm not sure. While you can configure the dataspace name, it probably won't fit universally. I opted for calling it just "Portal" for the white-label variant for now. Maybe "Authority Portal" is an alternative.

But I don't really know what's best in this case. Maybe @AbdullahMuk can give some input